<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="button_home">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_learn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Learn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_practice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Practice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_stats">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Statistics</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_games">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Games</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stack_main">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="LearnPage" name="page_learn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="HomePage" name="page_home">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="PracticePage" name="page_practice">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="TStatistics" name="page_stats">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="GamePage" name="page_games">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="action_change_user"/>
    <addaction name="action_homepage"/>
    <addaction name="action_option"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_aboutQt"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionChange_user">
   <property name="text">
    <string>Change user</string>
   </property>
  </action>
  <action name="actionOnline_competitoin">
   <property name="text">
    <string>Online Competition</string>
   </property>
  </action>
  <action name="actionOthers">
   <property name="text">
    <string>Others</string>
   </property>
  </action>
  <action name="actionText_practice">
   <property name="text">
    <string>Text practice</string>
   </property>
  </action>
  <action name="actionMap_keyboard">
   <property name="text">
    <string>Map keyboard</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_change_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change user</string>
   </property>
  </action>
  <action name="action_homepage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_option">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="action_aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>Pages/homepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LearnPage</class>
   <extends>QWidget</extends>
   <header>Pages/learnpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PracticePage</class>
   <extends>QWidget</extends>
   <header>Pages/practicepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GamePage</class>
   <extends>QWidget</extends>
   <header>Pages/gamepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TStatistics</class>
   <extends>QWidget</extends>
   <header>Pages/tstatistics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>button_home</tabstop>
  <tabstop>button_learn</tabstop>
  <tabstop>button_practice</tabstop>
  <tabstop>button_stats</tabstop>
  <tabstop>button_games</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
